code/script.js

```diff
@@ -1,19 +1,138 @@
+    1 - Carbonara
+    2 - Mediterranian
+    3 - Bolognese
+    Please enter the number of your choice:`);
+    typeOfName = ["Carbonara", "Mediterranian", "Bolognese"];
+    break;
+  case "Salad":
+    subType = prompt(`Select a Salad:
+    1 - Cesar
+    2 - Greek 
+    3 - Fruit
+    Please enter the number of your choice:`);
+    typeOfName = ["Cesar", "Greek", "Fruit"];
+    break;
+}
+
+// Alertmessage: Confirm the selceted subType option.//
+// Had trouble understanding the typeOfName 0,1,etc.. found that others had the same issues and copied the help they got - still dont fully understand it//
+if (subType === "1") {
+  alert(
+    `Great Choice, You've chosen to order ${typeOfName[0]} ${selectedChoice}!`
+  );
+} else if (subType === "2") {
+  alert(
+    `Great Choice, You've chosen to order ${typeOfName[1]} ${selectedChoice}!`
+  );
+} else if (subType === "3") {
+  alert(
+    `Great Choice, You've chosen to order ${typeOfName[2]} ${selectedChoice}!`
+  );
+}
+else {
+  alert(`Invalid choise - please select a number between 1-3 to make an order.`)
+}
+//STEP 3 END//
+
+
+
+// STEP 4 - Age//
+//Ask user for their age and confirm whether the order is selected for adult or child. Ask user to confirm.//
+// Promptmessage: Asking the user to type their age://
+let age = prompt("Please enter your age to let us know wheter you are ordering a childrens portion or an adult portion.");
+
+//Confrim the selected food size and price (adult or child portion) based on the user's choice.//
+// Have had some trouble linking to the correct terms - propably lack of understanding in terms of what the different functions are, had to guess in the end and do not understand the logic//
+if (age <= 18) {
+  alert(`You have selected a childrens portion of ${selectedChoice} and ${typeOfName[parseInt(subType) - 1]} to the cost of €15.`);
+} else if (age >= 18) {
+  alert(`You have selected an adult portion of ${selectedChoice} and ${typeOfName[parseInt(subType) - 1]} to the cost of €25.`);
+} else {
+  //Invalid choice - Errormessage://
+  alert(`Invalid choice. Please select a valid number.`);
+}
+// STEP 4 END //
 
-// Step 2 - Food choice
-// Your code goes here
 
-// Step 3 - Subtype choice
-// Your code goes here
+// STEP 5 - Order confirmation//
+//if the user confirms, a thank you message should be displayed using the alert() method, indicating that their meal will be prepared. If the user declines, a polite farewell message should be displayed using the alert() method, encouraging them to return for future orders. //
 
-// Step 4 - Age
-// Your code goes here
+let order = prompt(`Do you wish to place your order? Please write yes or no.`);
+if (order === "yes") {
+  alert(`Thank you! Your order is received and will be ready shortly.`);
+}
+else if (order === "no") {
+  alert(`Your order is canceled. We hope to see you again soon.`);
+}
+else {
+  alert(`Ìnvalid choice.`);
+}
 
-// Step 5 - Order confirmation
-// Your code goes here
+//STEP 5 END //
```

Comment:
Overall, I like how you comment on your code. It is descriptive !! I think it helps to understand your code better. I cannot write comments well, so I learned a lot from reading them in your code. Thank you!!